docs/manual/src/README.md

```diff
@@ -0,0 +1,57 @@
+# The docs for the "manual".
+
+The manual is what we publish to https://mozilla.github.io/uniffi-rs/ and
+is generated from the content in this directory.
+
+This README is not published as part of the docs - but almost everything
+else in this directory is. This document describes how that publishing
+works.
+
+* This repo uses `mkdocs` - but **we must never deploy using that tool**.
+  Deploying using mkdocs will publish into the root of the site, **damaging
+ the versioning system we have in place.**
+
+* We use a tool, [`mike`](https://github.com/jimporter/mike), which manages
+  the `mkdocs` build process and the deployment - it deploys to a versioned
+  (eg, `./0.27`) directory and manages aliases (eg, `latest`) on the site.
+
+* There are various HTML redirects and symlinks keeping things together. These
+  are maintained by hand. See also any docs related scripts in `./tools`.
+
+To put it another way: we never deploy the entire site, but instead use `mike`
+to deploy just a single version.
+
+## How to deploy
+
+* Prepare your Python environment by executing `pip install -r tools/requirements_docs.txt`
```

Comment:
If this is not inline code but a block it might be easier to copy (double click) vs needing to carefully select the text.